src/transport/raw/tests/NetworkTestHelpers.h

```diff
@@ -68,9 +68,16 @@ class LoopbackTransportDelegate
     // configurable allowed number of messages (mNumMessagesToAllowBeforeDropping)
     virtual void OnMessageDropped() {}
 };
-
 class LoopbackTransport : public Transport::Base
 {
+private:
+    // Use unique pointers for work callbacks, so that one callback does not cancel another.
```

Comment:
This is a workaround for the real issue, which is that ScheduleWork on the select impl is broken.  We should be fixing that, not working around it.

Reply:
Yes... did not try to yak shave into that one...